api/src/mailbox.rs

```diff
@@ -1059,8 +1059,8 @@ impl AuthAndStashFlags {
 #[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
 pub struct AuthorizeAndStashReq {
     pub hdr: MailboxReqHeader,
-    pub metadata: [u8; 4],
-    pub measurement: [u8; 48],
+    pub metadata: [u8; 4],     // Firmware Id.
```

Comment:
show we just rename this as fw_id? so as not to confuse with metadata in "image metadata entry"

Reply:
Makes sense, updated.